.spelling

```diff
@@ -200,6 +200,7 @@ shortcodes
 stateful
 stderr
 triaged
+triaging
```

Comment:
```suggestion
```

oh this is no longer going to be necessary with the suggested change above, although it might not know the word "triage" either and so might need replacing with that